infrastructure-playbooks/purge-docker-cluster.yml

```diff
@@ -521,26 +391,13 @@
         enabled: no
       failed_when: false
 
-    - name: remove node-exporter container
-      docker_container:
-        name: node_exporter
-        state: absent
-      failed_when: false
-
     - name: remove node_exporter service file
       file:
         name: /etc/systemd/system/node_exporter.service
         state: absent
 
     - name: remove node-exporter image
-      docker_image:
-        image: "{{ ceph_docker_registry }}/prom/node-exporter"
-        state: absent
-        force: yes
-      tags:
-        - remove_img
-      failed_when: false
-
+      command: "{{ container_binary }} rmi {{ node_exporter_container_image }}"
```

Comment:
should we add the remove_img tag ?

Reply:
I'm unsure this would work anyway in fact

Reply:
that's what I'm currently using to avoid to download the image again after the purge playbook.